src/integrationTest/java/ch/admin/bar/siard2/cmd/mssql/issues/siardsuite115/VarCharTypesIT.java

```diff
@@ -20,42 +22,72 @@ public class VarCharTypesIT {
     public SiardArchivesHandler siardArchivesHandler = new SiardArchivesHandler();
 
     @Rule
-    public MSSQLServerContainer<?> emptyDb = new MSSQLServerContainer<>(DockerImageName.parse("mcr.microsoft.com/mssql/server:2017-CU12"))
-            .acceptLicense();
-
-    @Rule
-    public MSSQLServerContainer<?> customDb = new MSSQLServerContainer<>(DockerImageName.parse("mcr.microsoft.com/mssql/server:2017-CU12"))
+    public MSSQLServerContainer<?> db = new MSSQLServerContainer<>(DockerImageName.parse("mcr.microsoft.com/mssql/server:2017-CU12"))
             .acceptLicense()
             .withInitScript(SqlScripts.MsSQL.SIARDSUITE_115);
 
-    //Assert that siard archive created by siardcmd is uploaded back to db
     @Test
-    public void uploadCreatedArchive_expectNoExceptions() throws SQLException, IOException, ClassNotFoundException {
-        val actualArchive = siardArchivesHandler.prepareEmpty();
+    public void downloadArchive_expectNoExceptions() throws SQLException, IOException, ClassNotFoundException {
```

Comment:
The method name is now misleading - there are actual assertions now!

Maybe just change it to `downloadArchive`